README.md

```diff
@@ -1,24 +1,43 @@
-# README
+# Project: Budjet App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Built With
+- Languages: Ruby
+- Framework: Ruby on rails
 
-Things you may want to cover:
+## Getting started
+To get a local copy of this exercice, Please follow these simple example steps.
+### Prerequisites
+- Postgresql
+- Ruby runtime environment
+- Rails framework
+#### 1. Clone this repository or download the Zip folder:"
 
-* Ruby version
+```bash command
+$ git clone https://github.com/benshidanny11/budget-app.git
+```
+#### 2. Navigate to the location of the folder in your machine:
+```bash command
+you@your-Pc-name:~$ cd <blog-app>
+```
+#### 3. Press Enter to navigate to your local clone.
 
-* System dependencies
+#### 4. Create database in postgresql named blog_rails_development.
 
-* Configuration
+#### 5. Run rake db:migrate to create migrations
 
-* Database creation
 
-* Database initialization
+👤 **Daniel Urimubenshi**
 
-* How to run the test suite
+- GitHub: [@benshidanny11](https://github.com/bensidanny11)
+- Twitter: [@DBenshi](https://twitter.com/DBenshi)
+- LinkedIn: [Daniel Urimubenshi](https://www.linkedin.com/in/danielurimubenshi/)
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🤝 Contributing
 
-* Deployment instructions
+Contributions, issues, and feature requests are welcome!
```

Comment:
* Kindly add to your README  an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). [Check requirement here ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.)